### PR TITLE
[ADD][16.0] product_packaging_usability module

### PR DESCRIPTION
**Pull request by bealdav**
im1

um2

im3

### CODE REVIEW

File: product_packaging_usability/models/product_packaging.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+from odoo import models
+
+
+class ProductPackaging(models.Model):
+    _inherit = "product.packaging"
+
+    def ui_goto_packaging_view(self):
+        self.ensure_one()
+        action = self.env.ref("product.action_packaging_view")._get_action_dict()
```
Comment (rousseldenis) [inline product_packaging_usability/models/product_packaging.py]: _for_xml_id() is better (check in code)

File: product_packaging_usability/tests/test_module.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+class Test(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+    def test_action(self):
+        product = self.env.ref("product.product_product_6")
+        action = product.ui_goto_packaging_view()
```
Comment (rousseldenis) [inline product_packaging_usability/tests/test_module.py]: Great! Like it. We are so often too lazy to do kind of tests. :smiley:

Comment (bealdav) [reply product_packaging_usability/tests/test_module.py]: I'm lazy (permanently) and now I am ashamed to write this kind of test

File: product_packaging_usability/models/product_packaging.py (hunk 1 above)
Comment (bealdav) [reply product_packaging_usability/models/product_packaging.py:11]: Not sure to understand, you say second is better ?
```
self.env.ref("product.product_product_6")

self.env["product.product"]._for_xml_id("product.product_product_6")
```

I think first is better

`self.env.ref` is used in whole odoo src

Comment (bealdav) [reply product_packaging_usability/models/product_packaging.py]:
```
action = self.env.ref("product.action_packaging_view")._get_action_dict()

action = self.env["ir.actions.act_window"]._for_xml_id(
            "product.action_packaging_view"
        )

```

I prefer first but ok

**Review by PaulGoubert — APPROVED** [review]
Functionnal review done.   LGTM.   Thank you @bealdav

**Review by mourad-ehm — APPROVED** [review]
Code review

**Comment by bealdav** [comment]
Please @rousseldenis could you merge ?

Thanks

**Pull request merged**